utils/_context/containers.py

```diff
@@ -473,3 +481,34 @@ def __init__(self, host_log_folder) -> None:
             host_log_folder=host_log_folder,
             healthcheck={"test": "/healthcheck.sh", "retries": 60},
         )
+
+
+class OpenTelemetryCollectorContainer(TestedContainer):
+    def __init__(self, host_log_folder) -> None:
+        super().__init__(
+            image_name="otel/opentelemetry-collector-contrib:latest",
+            name="collector",
+            command="--config=/etc/otelcol-config.yml",
+            environment={
+                "DD_API_KEY": os.environ.get("DD_API_KEY_3"),
```

Comment:
Move this line inside `configure` (it works as Scenario.configure : called only if it will be used, where init is always called).